packages/react-components/src/organisms/__tests__/UserNavigation.test.tsx

```diff
@@ -42,3 +43,17 @@ it('applies the passed href', () => {
     ),
   ).toHaveAttribute('href', '/profile');
 });
+
+it('enables My team link when user is onboarded', () => {
+  const { getAllByText, rerender } = render(
+    <UserNavigation {...props} userOnboarded={false} />,
```

Comment:
You should put the team within this test. We try to use the generic props to meet the minimum conditions and everything that is asserted on is overwritten within the test to try and avoid changing tests in future.

Reply:
```suggestion
    <UserNavigation {...props} teams={[{ name: 'Team 1', href: '/team-1' }]} userOnboarded={false} />,
```